Library/Formula/kafka.rb

```diff
@@ -14,7 +14,7 @@ class Kafka < Formula
 
   depends_on "gradle"
   depends_on "zookeeper"
-  depends_on :java => "1.7+"
+  depends_on :java => "1.7"
```

Comment:
It runs on Java 1.8 but since Kafka does not currently specify a sourceCapability version inside the build.gradle build script it is built with whatever is the current installed VM version.  I have generated  the following issue in Kafka https://issues.apache.org/jira/browse/KAFKA-2034 to have it explicitly set the sourceCapability version so this is not an issue.  
This means when I try and run locally and I only have Java 1.7 installed on my mac the product will not run, with the 'pre bottled version'.  If I do not use the pre-built (bottled) version everything works just fine.  If there is another way to capture such as passing in the sourceCapability version into the gradle build as 1.7 for the bottle I will do that.